rest/route/annotations.go

```diff
@@ -255,7 +255,7 @@ func (h *annotationByTaskPutHandler) Parse(ctx context.Context, r *http.Request)
 	}
 
 	// check if the task exists
-	t, err := task.FindOne(task.ById(h.taskId))
+	t, err := task.FindByIdExecution(h.taskId, h.annotation.TaskExecution)
```

Comment:
h.annotation is nil at this point, since we don't read it in until 275

Reply:
yeah I just realized that, my bad. I'm working on a fix.